### PR TITLE
Sync head

### PR DESCRIPTION
**Pull request by shayne-fletcher-da**
Sync to `f2efb6df52081e76f9336dbe1e56c83bb5573194`.

### CODE REVIEW

File: .ghci [hunk 1]
```diff
@@ -1,4 +1,5 @@
 :set -hide-package=ghc-boot-th
+:set -package=ghc-lib-parser
```
Comment (cocreature) [inline .ghci]: Why did this change now?

Comment (shayne-fletcher-da) [reply .ghci]: I didn't make that change. Neil did I guess. Could it relate to the fact that `exposed=False` now?

Comment (shayne-fletcher-da) [reply .ghci]: Screen Shot 2020-01-09 at 12 55 51 PM

Comment (cocreature) [reply .ghci]: ah yeah that’s it, thanks for the explanation!

**Pull request merged**